era-compiler-solidity/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "era-compiler-solidity"
+version = "1.5.2"
+authors = [
+    "Oleksandr Zarudnyi <a.zarudnyy@matterlabs.dev>",
+]
+license = "MIT OR Apache-2.0"
```

Comment:
I believe `license` and `edition` fields can go to the workspace level to not be duplicated in packages? Or we expect different ones?

Reply:
Usually we keep those on crate level to avoid copy-pasting later.
@sayon you may add yourself here :)

Reply:
Here is an example:
https://github.com/matter-labs/era-compiler-common/blob/main/solc-downloader/Cargo.toml#L4-L6
https://github.com/matter-labs/era-compiler-common/blob/main/Cargo.toml